Technical-Steering-Committee/principles.md

```diff
@@ -0,0 +1,43 @@
+# Principles
```

Comment:
Shouldn't we talk here about licenses and why/what licenses in particular are used/allowed within Zowe?

Reply:
No. This is covered in charter and the answer is very straightforward. We are just adhering to Linux Foundation policy.